backend/compact-connect/lambdas/nodejs/tests/lambda.test.ts

```diff
@@ -148,6 +161,7 @@ describe('Nightly runs', () => {
         // Verify an event report was sent
         expect(mockSendReportEmail).toHaveBeenCalled();
         expect(mockSendAllsWellEmail).not.toHaveBeenCalled();
+        
```

Comment:
Did you intend to check something here, or is this just a stray new line?

Reply:
Stray line, I've removed it.